app/controllers/shipit/shipit_controller.rb

```diff
@@ -10,12 +10,14 @@ class ShipitController < ApplicationController
 
     helper Shipit::Engine.routes.url_helpers
     include Shipit::Engine.routes.url_helpers
+    include ActionView::Helpers::DateHelper
```

Comment:
Please don't include helpers in controller. Use `self.class.helpers.time_ago_in_words`.